### DISCUSSION TITLE
PytorchClassifier is not trained from examples/get_started/pytorch.py

### DISCUSSION THREAD

**Discussion by brrbaral** [Trusted-AI/adversarial-robustness-toolbox#2056] [Q&A]
Hi, 
ART team, 
I was trying to play with the examples included in examples. I tried to started with Pytorch. I got error  as : 

TypeError: cross_entropy_loss(): argument 'input' (position 1) must be Tensor, not NoneType

I was trying to execute the file : https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/examples/get_started_pytorch.py

Pytorch version: 1.13.1+cu116
Platform: google colab

Thank You.

**Comment by beat-buesser** [comment]
Hi @brrbaral I'm not able to reproduce your error, can you provide more information?
